gst/nnstreamer/tensor_transform/tensor_transform.c

```diff
@@ -586,16 +589,37 @@ gst_tensor_transform_set_option_data (GstTensorTransform * filter)
         str_op = g_strsplit (str_operators[i], ":", -1);
         num_op = g_strv_length (str_op);
 
+        /* check whether per-channel */
+        if (g_ascii_strcasecmp (str_op[0], "per-channel") == 0) {
```

Comment:
move this into the loop (line 610) to prevent svace issue: (str_op[0] not null)

```
if (str_op[0]) {
  gchar **values = g_strsplit (str_op[1], "@", -1);
  guint num_values = g_strv_length (values);

  /* check whether per-channel */
  if (g_ascii_strcasecmp (str_op[0], "per-channel") == 0) {
    if (num_values > 1 && g_ascii_strcasecmp (values[0], "true") == 0) {
      .....
    }

    g_strfreev (values);
    continue;

    op_s = g_new0 (tensor_transform_operator_s, 1);
    ....
  }
}
```

Reply:
Thanks for pointing out. I modified the code and added a missing g_strfreev. PTAL